rocketchat_API/APISections/im.py

```diff
@@ -19,6 +19,10 @@ def im_create(self, username, **kwargs):
         """Create a direct message session with another user."""
         return self.call_api_post("im.create", username=username, kwargs=kwargs)
 
+    def im_create_multiple(self, usernames, **kwargs):
```

Comment:
I think it'd be better if we use a list here instead of a string with the commas inside. We can then construct the string using `join`. I think it'll be more user-friendly.

Reply:
Sure - will do! Where you able to get the breaking test running? I couldn't figure out why it gets broken :(

Reply:
Looks like the API does no longer allows updating the user email by default. I removed the failing test and will bring it on to the devs. The fix is on master already. You can rebase to test.